function/value.go

```diff
@@ -34,12 +34,17 @@ type Value interface {
 }
 
 type conversionError struct {
-	from string
-	to   string
+	from  string
+	to    string
+	value interface{}
 }
 
 func (e conversionError) Error() string {
-	return fmt.Sprintf("cannot convert from type %s to type %s", e.from, e.to)
+	return fmt.Sprintf("cannot convert %+v (type %s) to type %s", e.value, e.from, e.to)
```

Comment:
Yeah, I could just change it to a string and add a `fmt.Sprintf("%f")` for the scalars.